pyglmnet/pyglmnet.py

```diff
@@ -232,7 +169,7 @@ def copy(self):
         """Return a copy of the object."""
         return deepcopy(self)
 
-    def qu(self, z):
+    def __qu(self, z):
```

Comment:
do I see a double underscore? @pavanramkumar I think you need only a single underscore